lib/configuration/configuration.js

```diff
@@ -1,8 +1,9 @@
 const yaml = require('js-yaml')
 const _ = require('lodash')
-const NodeCache = require('node-cache')
+const CacheManager = require('../cache/cache')
 
-const configCache = new NodeCache({ stdTTL: 0, maxKeys: 5000 })
+// Setup the cache manager
+var cacheManager = new CacheManager()
```

Comment:
can we please use either `const` or `let` instead of `var` for consistency sake?